defaults/main/bot-lists.yml

```diff
@@ -0,0 +1,49 @@
+---
+nginx_global_robots_txt:
+  - "User-agent: *"
+  - "Crawl-delay: 60"
+#  - "# Example 1: Block only Googlebot"
+#  -  "User-agent: Googlebot"
+#  -  "Disallow: /"
+#  - "# Example 2: Block Googlebot and Adsbot"
+#  -  "User-agent: Googlebot"
+#  -  "User-agent: AdsBot-Google"
+
+nginx_bots_deny_list:
+  - baidu
+  - ahrefs
+  - yandex
+  - ezooms
+  - yahoo
+  - webmeup-crawler
+  - semrush
+  - mj12bot
+  - dotbot
+  - 360spider
+  - pagefreezer
+  - Dorkbot
+  - semanticscholar
+  - aspiegel
+
+nginx_bots_request_limit_user_agents_regex:
```

Comment:
Could this be added in a separate file? I don't truly undertand why the previous list has only names, and this has a huge list of regexp. Is performance affected by using bigger regexps, instead of more entries?

Reply:
nginx limits the regex lines, that's the reason because we are using several regex lines.

Reply:
Would it make sense to have a regex for banned_bots, and keep the ratelimited_bots as is?

Reply:
We can use a single line with:

```
nginx_bots_deny_list:
  - (baidu|ahrefs|yandex|ahrefs|yandex|ezooms|yahoo|webmeup-crawler|semrush|mj12bot|dotbot|360spider|pagefreezer|Dorkbot|semanticscholar|aspiegel)
```

But we are going to loose the capacity of adding a comment for every banned header regex. Probably is useful this comment to justify the 403 status.

Right now this is a short list, so probably is more clear to use 1 item per bot, but for me is pretty the same.

Reply:
Again, wouldn't it be easier to maintain an allow list of browser user-agents, than a deny list of bot user-agents?  I would guess there's a lot less browser user-agent strings.

Reply:
It is probably the easiest and more efficient, but then we are forcing the user to use certain web browsers or their user-agents